dashboard/app/views/sessions/lockout.html.haml

```diff
@@ -0,0 +1,12 @@
+#lockout-container{
+  :"data-request-date" => @request_date.iso8601,
+  :"data-delete-date" => @delete_date.iso8601,
+  :"data-pending-email" => @pending_email,
+  :"data-api-url" => url_for(
```

Comment:
- https://github.com/rubocop/ruby-style-guide#symbols-as-keys
- https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#ruby-hashes

Reply:
yep I've already made these changes a few days ago. i forgot to push them up to this branch.